charts/authentik/templates/_helpers.tpl

```diff
@@ -14,6 +14,17 @@ Create authentik server worker and version as used by the chart label.
 {{- printf "%s-%s" (include "authentik.fullname" .) .Values.worker.name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Determine the namespace to use, allowing for a namespace override.
```

Comment:
This also needs to go into the `authentik-remote-cluster` chart (probably named `authentik-remove-cluster.namespace`)

Reply:
Thank you for the hint. I addressed this in [575d670](https://github.com/goauthentik/helm/pull/316/commits/575d6700fc47d7d150c4371317d23d0769ac0924)